setup.py

```diff
@@ -9,7 +9,7 @@
     'luigi',
     'boto3',
     'slack_sdk',
-    'pandas',
+    'pandas<1.2',
```

Comment:
This is needed because pandas 1.2 has breaking change. I'll (or other super engineers will) fix this.